libraries/AP_HAL_SITL/SITL_State.cpp

```diff
@@ -376,6 +377,10 @@ void SITL_State::_fdm_input_local(void)
         _output_to_flightgear();
     }
 
+#if HAL_GCS_ENABLED
+    GCS::get_singleton()->send_message(MSG_SIM_STATE);
```

Comment:
not sure about this one as this means to trigger the mavlink sending at high rate. There is an interest to have this but probably not in the general case. 
Maybe we could put this under a SIM_HIGH_RATE_STATE_MSG .

I have not checked the implication of loading GCS here, but on SITL on HW that could also be very bad.

Reply:
I think the queued messages won't send any faster than 50hz.